controller/service.rst

```diff
@@ -104,8 +104,9 @@ defined above with the id ``app.hello_controller``::
 
```

Comment:
Unrelated, but I think we should remove the `forward` example entirely and just keep the routing example (and probably move the note below the routing example).